File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    let maxWords=0;
    for(let i=0;i<sentences.length;i++){
           let eachSentenceWordsCount = sentences[i].split(" ").length;
           maxWords= Math.max(eachSentenceWordsCount,maxWords);
        }
        return maxWords;    
};

File: 2219-maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int maxLength=Integer.MIN_VALUE;
        for(String sentence:sentences){
            String[] words = sentence.trim().split(" ");
            int length =words.length;
            if(length>maxLength){
                maxLength=length;
            }
        }

        return maxLength;
    }
}